tests/mocks/drawer.ts

```diff
@@ -0,0 +1,35 @@
+const drawerLinks = [
```

Comment:
Ditto: I am a bit unclear why we need to mock this while they don't come via an API and they just get rendered by Frontend?